tsconfig.base.json

```diff
@@ -16,10 +16,10 @@
     ],
     "esModuleInterop": true,
     "alwaysStrict": true,
-    "strictNullChecks": false, // TODO improve linting with this set to true
-    "noImplicitAny": false, // TODO improve linting with this set to true
```

Comment:
Seems like a good idea to me to warn for implicit `any` types.

Reply:
Yeah this is the plan, but I didn't want to spam this PR with all those linting fixes. So will make a separate PR where I'm just tightening up linting.

Reply:
I meant removing the TODO, since it's a good reminder...

Reply:
Yeah, there was a linting issue with comments in a JSON file I think, I moved them to this issue instead: :) https://github.com/sockethub/sockethub/issues/707